src/components/ItemList/ItemRow.tsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
-import DatalistItem from 'box-ui-elements/es/components/datalist-item';
+import classNames from 'classnames';
 import { UserMini, GroupMini } from '../../@types';
 import './ItemRow.scss';
 
 export type Props = {
     id?: string;
+    isActive?: boolean;
     item?: UserMini | GroupMini;
     name?: string;
 };
 
-const ItemRow = ({ item, ...rest }: Props): JSX.Element | null => {
+const ItemRow = ({ isActive, item, ...rest }: Props): JSX.Element | null => {
     if (!item || !item.name) {
         return null;
     }
 
     return (
-        <DatalistItem {...rest}>
+        <li {...rest} className={classNames('ba-ItemRow', { 'is-active': isActive })} data-testid="ba-ItemRow">
```

Comment:
Is this going to dump any non-DOM props onto a DOM element?

Reply:
Seems to be missing event handling and `role="option"` that's in BUIE, as well. Does this work as expected?

Reply:
> Seems to be missing event handling and `role="option"` that's in BUIE, as well. Does this work as expected?

Those are for ARIA. It works as expected on my devvm.